lib/connection/connection.js

```diff
@@ -103,54 +103,39 @@ function Connection(context)
     return id;
   };
 
-  this.heartbeat = function (self)
+  this.heartbeat = callback =>
   {
     Logger.getInstance().debug("Issuing heartbeat call");
-    var requestID = uuidv4();
+    const requestID = uuidv4();
 
-      const request = services.sf.request(
+    services.sf.request(
         {
           method: 'POST',
           url: Url.format(
-            {
-              pathname: '/session/heartbeat',
-              search: QueryString.stringify(
-                {
-                  requestId: requestID
-                })
-            }),
-          callback: function (err, body)
+              {
```

Comment:
Strange formatting - four spaces here, two spaces below... Also, opening parenthesis in separate lines, closing in the same?